### PR TITLE
updated taquito/beacon-wallet readme to match beacon implementation

### LINKED ISSUE #2908: [beacon-wallet] Update the README file (and documentation) to include an ACTIVE_ACCOUNT_SET listener
The latest versions of [@airgap/beacon-sdk](https://github.com/airgap-it/beacon-sdk) require an additional listener (`ACTIVE_ACCOUNT_SET`) that emits events when users switch accounts or disconnect their address on the wallet side.

The README.md file (in [packages/taquito-beacon-wallet/README.md](https://github.com/ecadlabs/taquito/blob/master/packages/taquito-beacon-wallet/README.md)) needs to match Beacon's documentation ([see taquito tab](https://docs.walletbeacon.io/getting-started/first-dapp)) which includes the following:
```ts
// Listen for all the active account changes
wallet.client.subscribeToEvent(
  BeaconEvent.ACTIVE_ACCOUNT_SET,
  async (account) => {
    // An active account has been set, update the dApp UI
    console.log(`${BeaconEvent.ACTIVE_ACCOUNT_SET} triggered: `, account);
  },
);
```

### PR DESCRIPTION
**Pull request by dsawali**
closes #2908 

Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have run the linter against the changes
- [ ] You have added unit tests (if relevant/appropriate)
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

In this PR, please also make sure: 

- [ ] You have linked this PR to the issue by putting `closes #TICKETNUMBER` in the description box (when applicable)
- [ ] You have added a concise description on your changes
## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: packages/taquito-beacon-wallet/README.md [hunk 1]
```diff
@@ -26,23 +26,27 @@ import { BeaconWallet } from '@taquito/beacon-wallet';
 const options = {
   name: 'MyAwesomeDapp',
   iconUrl: 'https://taquito.io/img/favicon.svg',
-  preferredNetwork: 'chosen_network',
+  network: { type: 'ghostnet' },
   eventHandlers: {
```
Comment (hui-an-yang) [inline packages/taquito-beacon-wallet/README.md:30]: wasn't sure how relevant is `eventHandlers` now there is `subscribeToEvent` in place

File: packages/taquito-beacon-wallet/README.md [hunk 2]
```diff
@@ -26,23 +26,27 @@ import { BeaconWallet } from '@taquito/beacon-wallet';
...
 };
 const wallet = new BeaconWallet(options);
 
-// The Beacon wallet requires an extra step to set up the network to connect to and the permissions:
-await wallet.requestPermissions({
-  network: {
-    type: 'chosen_network',
+await wallet.requestPermissions();
+
+wallet.client.subscribeToEvent(
```
Comment (hui-an-yang) [inline packages/taquito-beacon-wallet/README.md:43]: looks like `subscribeToEvent` is returning a `Promise<void>`,  would be safe to add `await` in front

**Review by hui-an-yang — CHANGES_REQUESTED** [review]
Left a couple comments in filesAlso, would be good to update `docs/originate.md` line 51 with `subscribeToEvent`  too (also 19.2.0 versioned doc). Our website ins't using subscribeToEvent either and getting warning in console with `An active account has been received, but no active subscription was found for BeaconEvent.ACTIVE_ACCOUNT_SET.
        For more information, visit: https://docs.walletbeacon.io/guides/migration-guide`

**Comment by dsawali** [comment]
> Left a couple small comments in files Also, would be good to update `docs/originate.md` line 51 with `subscribeToEvent` too (and 19.2.0 versioned doc). Our website ins't using `subscribeToEvent` either and getting warning in console with `An active account has been received, but no active subscription was found for BeaconEvent.ACTIVE_ACCOUNT_SET. For more information, visit: https://docs.walletbeacon.io/guides/migration-guide`

updating originate.md is a good idea, but I will not be updating the website in this ticket. We can create a ticket and request @ac10n's help to implement the change

File: packages/taquito-beacon-wallet/README.md (hunk 2 above)
Comment (hui-an-yang) [reply packages/taquito-beacon-wallet/README.md]: they also don't have `await` in front of `requestPermission() either`.  And i think dealing with promises in sequence is better to `await` each other out

Comment (hui-an-yang) [reply packages/taquito-beacon-wallet/README.md]: On that note, I think it's supposed to `subscribeToEvent` first then `requestPermission` like their example

**Review by hui-an-yang — APPROVED** [review]
Ready to merge once the order is updated, thanks for prompt update!

**Pull request merged**